javascript/javascript2/week1/movieApp/index.js

```diff
@@ -1,259 +1,141 @@
-const mainApp = document.getElementById("root");
-const header = document.createElement("header");
-header.setAttribute("class", "headerContainer");
-const h1 = document.createElement("h1");
-h1.innerText = "MovieApp";
-header.appendChild(h1);
-const navLinkList = ["Home", "Collections", "Contact Us"];
-/* common Movie Card
- */
-function getMovieCard(movie) {
-  const cardContainer = document.createElement("div");
-  cardContainer.setAttribute("class", "cardContainer fade-animation");
-
-  /* image */
-  const heroImage = document.createElement("img");
-  heroImage.setAttribute("class", "heroImage");
-
-  heroImage.src = movie.poster_url;
-  cardContainer.appendChild(heroImage);
+const timingSection = document.createElement("section");
+const timingTitle = document.createElement("h2");
+timingTitle.innerText = "Your timing in in Our App";
+const timingPara = document.createElement("p");
+timingPara.setAttribute("class", "timingPara");
+
+const newPar = document.createElement("p");
+newPar.textContent = "00:00:00";
+newPar.setAttribute("class", "timingPara");
+
+const newPara2 = document.createElement("p");
+newPara2.setAttribute("class", "newPara2");
+newPara2.innerText =
+  "Lets Check how much you spend on Selecting Movie, Please Press Start Button and go on movie to select any of them";
+
+const startTimer = document.createElement("button");
+startTimer.innerText = "Start";
+startTimer.setAttribute("class", "primaryButton");
+const stopTimer = document.createElement("button");
+stopTimer.innerText = "Stop";
+stopTimer.setAttribute("class", "primaryButton");
+
+const resetTimer = document.createElement("button");
+resetTimer.innerText = "Reset";
+resetTimer.setAttribute("class", "primaryButton");
+
+timingSection.append(
+  timingTitle,
+  timingPara,
+  newPara2,
+  newPar,
+  startTimer,
+  stopTimer,
+  resetTimer
+);
+startTimer.addEventListener("click", () => {
```

Comment:
Same comment. You don't need these arrow functions here.
```suggestion
startTimer.addEventListener("click", start);
```